types/wicg-native-file-system/index.d.ts

```diff
@@ -0,0 +1,155 @@
+// Type definitions for non-npm package Native File System API 2020.06
+// Project: https://github.com/WICG/native-file-system
+// Definitions by: Ingvar Stepanyan <https://github.com/RReverser>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 3.5
+
+interface FilePickerAcceptType {
+    description?: string;
+    accept: Record<string, string[]>;
```

Comment:
```suggestion
    accept?: Record<string, string[]>;
```

Should this be marked as optional? https://wicg.github.io/native-file-system/#native-filesystem

Reply:
I've intentionally marked this as required to prevent developers from accidentally passing in an empty object (which won't do anything). This deviates from the spec, but I suppose slightly more helpful typings-wise.